crates/bevy_asset/src/io/file/file_watcher.rs

```diff
@@ -166,72 +219,103 @@ pub(crate) fn new_asset_event_debouncer(
                                         },
                                     );
                                 } else {
-                                    match (old_is_meta, new_is_meta) {
-                                        (true, true) => {
+                                    // match (old_is_meta, new_is_meta) {
```

Comment:
Remember to uncomment this.

Reply:
Oh yeah, I need to finish that part of the feature.  I'll do that now. Almost forgot!